mlos_bench/db/db_schema_pg.sql

```diff
@@ -0,0 +1,108 @@
+--
+-- Copyright (c) Microsoft Corporation.
+-- Licensed under the MIT License.
+--
+-- PostgreSQL version of the DB schema.
+
+DROP TABLE IF EXISTS experiment_config CASCADE;
+DROP TABLE IF EXISTS experiment_merge CASCADE;
+DROP TABLE IF EXISTS trial_status CASCADE;
+DROP TABLE IF EXISTS trial_config CASCADE;
+DROP TABLE IF EXISTS trial_results CASCADE;
+DROP TABLE IF EXISTS trial_telemetry CASCADE;
+
+DROP TYPE IF EXISTS experiment_id_t CASCADE;
+DROP TYPE IF EXISTS trial_id_t CASCADE;
+DROP TYPE IF EXISTS param_id_t CASCADE;
+DROP TYPE IF EXISTS param_value_t CASCADE;
+DROP TYPE IF EXISTS status_t CASCADE;
+
+CREATE DOMAIN experiment_id_t AS VARCHAR(255) NOT NULL;
+CREATE DOMAIN trial_id_t AS INTEGER NOT NULL;
+CREATE DOMAIN param_id_t AS VARCHAR(255) NOT NULL;
+CREATE DOMAIN param_value_t AS VARCHAR(255);
+
+-- Should match the `mlos_bench.environment.Status` enum.
+CREATE TYPE status_t AS ENUM (
+    'UNKNOWN',
+    'PENDING',
+    'READY',
+    'RUNNING',
+    'SUCCEEDED',
+    'CANCELED',
+    'FAILED',
+    'TIMED_OUT'
+);
+
+CREATE TABLE experiment_config (
+    exp_id experiment_id_t,
+    descr TEXT,
+    git_repo VARCHAR(255) NOT NULL,
+    git_commit VARCHAR(40) NOT NULL,
+
+    PRIMARY KEY (exp_id)
+);
+
+CREATE SEQUENCE trial_id_seq;
```

Comment:
Also seems to be missing a `DROP SEQUENCE IF EXISTS`

Reply:
turned out, this sequence gets remove automatically because it is owned by a column. Anyway, I've decided to not use autoincrement for now and ditch all postgres-specific features in the first version for simplicity